securebanking-openbanking-uk-rcs-server/src/test/java/com/forgerock/securebanking/openbanking/uk/rcs/api/ConsentDetailsApiControllerTest.java

```diff
@@ -102,147 +107,69 @@ public DateTime deserialize(JsonElement json, Type typeOfT, JsonDeserializationC
     private ApiClientServiceClient apiClientService;
     @MockBean
     private UserServiceClient userServiceClient;
-
     @Autowired // needed for tests purposes
     private ConsentDetailsFactoryProvider consentDetailsFactoryLocator;
     @Autowired
     private TestRestTemplate restTemplate;
 
-    // ACCOUNT
-    @Test
-    public void ShouldGetAccountConsentDetails() throws ExceptionClient {
```

Comment:
Is there an AccountConsent test in the refactored code?

Reply:
I forget add arguments to test the account details and now all cases have coverage. Now the cast in the condition to set the debtor account have coverage too.
Fixed